packages/system/keycloak/templates/sts.yaml

```
@@ -0,0 +1,135 @@
+    spec:
+      restartPolicy: Always
+      securityContext:
+        fsGroup: 1000
+      containers:
+        - name: keycloak
+          image: {{ .Values.image }}
+          imagePullPolicy: Always
+         {{- if or .Values.resources.requests .Values.resources.limits }}
+          resources:
+            {{- if .Values.resources.limits }}
+            limits:
```

yamllint

[warning] wrong indentation: expected 10 but found 12

(indentation)